RetrievalAugmentedGeneration/common/utils.py

```diff
@@ -126,8 +126,8 @@ def get_vector_index() -> VectorStoreIndex:
 
     logger.info(f"Using {config.vector_store.name} as vector store")
     if config.vector_store.name == "pgvector":
-        connection_string = f"postgresql://{os.getenv('POSTGRES_USER', '')}:{os.getenv('POSTGRES_PASSWORD', '')}@{config.vector_store.url}"
-        db_name = "vector_db"
+        connection_string = f"postgresql://{os.getenv('POSTGRES_USER', '')}:{os.getenv('POSTGRES_PASSWORD', '')}@{config.vector_store.url}/{os.getenv('POSTGRES_DB', '')}"
+        db_name = os.getenv('POSTGRES_DB', '')
```

Comment:
You can use `db_name` to create `connection_str`. Also will recommend using some string as default db name instead of `''` as if someone didn't specify `POSTGRES_DB` environment variable, flow will not break. 
```
        db_name = os.getenv('POSTGRES_DB', 'vector_db')
        connection_string = f"postgresql://{os.getenv('POSTGRES_USER', '')}:{os.getenv('POSTGRES_PASSWORD', '')}@{config.vector_store.url}/{db_name}"
```

Reply:
Hi @nv-pranjald, thank you for your feedback! I have incorporated your comments into the relevant files. Please help review again!
Regarding the parameter 'AI_PLAYGROUND_API_KEY', it is utilized in the 'docker-compose-pgvector.yaml' file. I once encountered a situation where the 'AI_PLAYGROUND_API_KEY' definition was missing in the source environment file. I want to confirm that if it is not assigned a value, it will automatically be given a default value. Therefore, I have removed it.